app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -18,22 +18,10 @@
         v-on-clickaway="hideEmojiPicker"
         :on-click="emojiOnClick"
       />
-      <resizable-text-area
```

Comment:
We cannot use the rich content editor for Facebook/Twitter/Whatsapp as they don't support markdown format we use.